Support for alternative container runtimes - Podman (#2745)
dapr/docs: daprdocs/content/en/getting-started/install-dapr-selfhost.md

The recommended development environment requires [Docker](https://docs.docker.com/install/). While you can [initialize Dapr without a dependency on Docker]({{<ref self-hosted-no-docker.md>}})), the next steps in this guide assume the recommended Docker development environment.
{{% /alert %}}

> >> Note - You can also install [Podman](https://podman.io/) in place of Docker. Please check [Dapr init commands]({{<ref dapr-init.md>}}) for details on how to initialise Dapr using Podman.

Reviewer: Can you add this to the {{% alert %}} block above?

```suggestion
You can also install [Podman](https://podman.io/) in place of Docker. Read more about [initializing Dapr using Podman]({{<ref dapr-init.md>}}).
```
